src/main/java/org/opentripplanner/graph_builder/issue/api/DataImportIssueStore.java

```diff
@@ -29,6 +29,8 @@ public interface DataImportIssueStore {
    */
   void add(String type, String message, Object... arguments);
 
+  void setFilename(String filename);
```

Comment:
JavaDoc - see my comment below. Both methods need JavaDoc. I think we should be a bit more generic here, and we should also rename the methods so they indicate the intention. So, I suggest using:
```
void startProcessingSource(String source);
void stopProcessingSource();
```
An alternative is to return a `Closeable` from `startProcessingSource`. The JavaDoc on `stopProcessingSource` should remind the developers that it is mandatory and needs to be called in a finally block, and safe no try catch needed inside the finally.